### DISCUSSION TITLE
一个想法，NFT捐赠

### DISCUSSION THREAD

**Discussion by verbo2009** [rebase-network/hello-world#67] [Ideas]
NFT小白，开个脑洞
1.鉴于NFT的流动性特性，以及NFT的供给不断增加，很多人手头会有越来越多的“库存”NFT
2.如果有个捐赠协议，用户可通过捐赠NFT，获得协议发行的代币
3.协议再把收到的NFT以新的定价策略挂出，从而形成一个捐赠NFT二手市场，大家可以在这里“淘宝”
4.销售收入再用来对代币进行回购

**Comment by luckyyang** [comment]
就是一个交易市场？NFT 的定价是个问题
